woocommerce/woocommerce-ios
# Include build configuration details in all tracks events

Fixes #4432 

This PR is only to test and integrate the changes in https://github.com/Automattic/Automattic-Tracks-iOS/pull/182.

## To test

1. Check out https://github.com/woocommerce/woocommerce-ios/pull/4468 and run the Woo app.
2. You can set a breakpoint in `- [TracksServiceRemote sendBatchOfEvents:withSharedProperties:completionHandler:]` and ensure that the `properties` argument contains `Debug` for `device_info_app_build_configuration`.
3. Check the Tracks Live View after a few minutes, and verify that the events coming from the app include the new property.

Update release notes:
